start/resources/multicore-jobs.md

```diff
@@ -2,15 +2,20 @@
 
 [TOC]
 
-OSG has limited support for multicore jobs. These can be used for
-threaded or OpenMP applications. To request a multicore slot, simply use 
-the HTCondor *request_cpus* attribute in your submit file. Example:
+Please note, the OSG has limited support for multicore jobs. Multicore jobs
+can be submitted for threaded or OpenMP applications. To request multiple cores
+(aka cpus) use the HTCondor *request_cpus* attribute in your submit file. 
+
+Example:
 
     request_cpus = 8
 
-Multicore jobs can currently request up to 8 cores. Please note that
-you have to limit your code to the same number of cores. Do not use
-core auto-detection as it might detect cores not assigned to your job.
-If you request 8 cores from the scheduler, limit your code to 8
-threads as well.
+Currently, multicore jobs can request a maximum of 8 cpus. 
```

Comment:
@rynge is this true?

Reply:
We can do more than 8, but it is still uncommon to have those resources available at all times.

I think the auto-detection part is important. When you request something like 8, there is a good chance it is on a larger machine, and if you let the code auto detect it will start more threads than the 8 requested.

Reply:
Maybe change the language to say "we recommend a maximum of 8"